src/js/utils/browser.js

```diff
@@ -1,11 +1,14 @@
 import { head } from "lodash/fp"
+import remoteServices from "../remoteServices"
+
 export const isChrome = () => typeof browser === "undefined" && chrome
 export const isFirefox = () => typeof browser !== "undefined" && chrome
 
 const DEFAULT_SUBDOMAIN = "unset"
 
 export const getSettings = (withDefaultSubdomain = true) => {
-  const keys = ["subdomain", "apiKey", "settingTimeTrackingHHMM"]
+  const keys = ["subdomain", "apiKey", "settingTimeTrackingHHMM",
+    ...Object.values(remoteServices).map(remoteService => `hostOverrides:${remoteService.name}`)]
```

Comment:
Why are the hostOverrides not stored as an object in the settings, e.g.
```
{
  hostOverrides: { github: "https://yay.gitlab.com" }
}
```

Reply:
Could you try to store the hostOverrides in an object as suggested above instead?

Reply:
Because chrome storage can only handle an object with one level of nesting, unless that changed recently. I can check it once more to be sure, but in the past you could only pass an object of primitives, any objects below that would not be stored.

Reply:
Yeah, I thought that might be the reason and therefor tested that on Chrome and it worked to store an additional object under the key `hostOverrides`.

Reply:
Then that must have changed. Even better, that will make the code quite a bit cleaner :)

Reply:
Yes, exaclty, I think so too 👍🏼